src/Confluent.Kafka/Admin/ListOffsetsRequest.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Confluent.Kafka.Admin
+{
+    /// <summary>
+    ///     ListOffsetsRequest for making a call to ListOffsets.
+    /// </summary>
+    public class ListOffsetsRequest
```

Comment:
`ListOffsetsRequest` doesn't makes sense as we are passing a List of them in the `ListOffsetsAsync`. I think `TopicPartitionOffsetSpec` would be more relevant.

Reply:
Okay will make the change!